mcp-test.py

```diff
@@ -34,10 +28,9 @@ def main(host: str, port: int):
     else:
         selected_model = available_models[0]
         print(f"Using model: {selected_model}")
-
     agent_config = AgentConfig(
         model=selected_model,
-        instructions="You are a helpful assistant",
+        instructions="You are a helpful assistant. When asked about weather, use the getforecast tool with latitude and longitude parameters.",
```

Comment:
I'd prefer not to explicitly mention the getforecast tool, because one of the benefits of MCP is the ability for the client to discover the available tools .

Reply:
@deewhyweb that is a great point! I was so focused last night on just getting it to 'work'.